README.rst

```diff
@@ -17,7 +17,3 @@ Kinto
 
 Kinto is a server allowing you to store and synchronize arbitrary data,
 attached to your Firefox account.
-
-* `Online documentation <http://kinto.readthedocs.org/en/latest/>`_
```

Comment:
the README is also displayed on the first page of the github repo. It's also used as the pypi page, I believe